doc/guides/device_mgmt/ota.rst

```diff
@@ -0,0 +1,83 @@
+.. _ota:
+
+Over-the-Air Update
+###################
+
+Overview
+********
+
+Over-the-Air (OTA) Update is a method for delivering firmware updates to remote
+devices using a network connection. Although the name implies a wireless
+connection, updates received over a wired connection (such as Ethernet)
+are still commonly referred to as OTA updates. This approach requires server
+infrastructure to host the firmware binary and implement a method of signaling
+when an update is available. Security is a concern with OTA updates; firmware
+binaries should be cryptographically signed and verified before flashing.
+
+The :ref:`dfu` section discusses upgrading Zephyr firmware using MCUboot. The
+same method can be used as part of OTA. The binary is first downloaded
+into an unoccupied code partition, then flashed using the `MCUboot bootloader`_
+process.
+
+Examples of OTA
+***************
+
+Golioth
+=======
+
+`Golioth`_ is an IoT management platform that includes OTA updates. Devices are
+configured to observe your available firmware revisions on the Golioth Cloud.
+When a new version is available, the device downloads and flashes the binary. In
+this implementation, the connection between cloud and device is secured using
+TLS/DTLS, and the signed firmware binary is confirmed by MCUboot before flashing
+occurs.
+
+1. A working sample can be found on the `Golioth Zephyr-SDK repository`_
+2. The `Golioth OTA documentation`_ includes complete information about the
+   versioning process
+
+Eclipse hawkBit |trade|
+=======================
+
+`Eclipse hawkBit`_ |trade| is an update server framework that uses polling on a
+REST api to detect firmware updates. When a new update is detected, the binary
+is downloaded and MCUboot verifies the signature before flashing the firmware.
```

Comment:
```suggestion
is downloaded and installed. MCUboot can be used to verify the signature before flashing the firmware.
```

I think it's worth making some change along these lines because hawkbit itself doesn't require mcuboot; that's an implementation decision made by the sample. Feel free to reword if you want.

Reply:
Good suggestion, thanks.